nights/2024-05-13-README.md

```diff
@@ -18,17 +18,19 @@ Delete this list if there are no unusual images.
 
 Check these off if they are true:
 
-- [ ] No images are missing filter information (except BIAS and DARK, which need no filter).
-- [ ] No images are missing pointing information (RA/Dec and WCS)
-- [ ] No images are missing object names (only applies to science images)
-- [x] EXAMPLE checked-off box, please delete.
+- [x] No images are missing filter information (except BIAS and DARK, which need no filter).
+- [x] No images are missing pointing information (RA/Dec and WCS)
+- [x] No images are missing object names (only applies to science images)
+
 
 If any images are missing information and you have been unable to fix them please list
 them below with a short description of the problem.
 
-+ `m34-002R.fit` -- no WCS, looks like there were maybe clouds.
-+ `m404-001B.fit` -- Not sure what object this, and googling `m404` got me a 404.
-+ `sa113-099V.fit` -- No filter in the FITS header, not sure what it should be.
+ey-uma-S001-R002-C004-rp.fit - Satellite
+ey-uma-S001-R002-C005-rp.fit - Satellite
+ey-uma-S001-R002-C010-rp.fit - Satellite
+ey-uma-S001-R002-C014-rp.fit - Either two satellites or maybe a slight shift in the telescope. The satellites seem strangely parallel...
```

Comment:
Two satellites most likely. Star shapes are good enough that it wouldnt be a shift in the telescope, probably star links or some other satellite constellation.